src/cli/npc.rs

```diff
@@ -18,6 +18,7 @@ impl Display for CliError {
     }
 }
 
+#[serde(rename_all = "lowercase")]
 enum CliResult {
     Full(NetState),
     Ifaces(Vec<Iface>),
```

Comment:
I think this rename is not needed as this enum is not being serialize. What do you think? @cathay4t

Reply:
Ok, i'll make the changes.